pierone/cli.py

```diff
@@ -35,17 +35,6 @@ def print_version(ctx, param, value):
     ctx.exit()
 
 
-def validate_pierone_url(url: str) -> None:
-    ping_url = url.rstrip('/') + '/swagger.json'
-    try:
-        response = requests.get(ping_url, timeout=5)
```

Comment:
I believe it should have been `requests.head` instead.

Reply:
It should but we are removing it completly